extensions/amazon-sales/release-notes/index.md

```diff
@@ -0,0 +1,44 @@
+---
+group: extensions
+title: Amazon Sales Channel Release Notes
+---
+
+*Release notes published April 2019.*
+
+**Amazon Sales Channel** is now available with Magento 2.2.x and 2.3.x. Merchants can use Amazon Sales Channel to integrate with an Amazon Seller account to post and sell Magento products on Amazon.
```

Comment:
"Magento" products? 

Maybe this?

> "...post and sell products in your Magento catalog on Amazon."

Also, is this available for both Open Source and Commerce? If so, use similar language as line 24. If not, be specific and use a jekyll variable for the product name. See line 24.